lib/utils.bash

```diff
@@ -67,6 +67,23 @@ get_install_path() {
   fi
 }
 
+get_latest_version() {
+  # pattern from xxenv-latest (https://github.com/momo-lab/xxenv-latest)
+  local versions=$1
+  versions=$(echo "$versions" |
+    grep -vE "(^Available versions:|-src|-dev|-latest|-stm|[-\.]rc|-alpha|-beta|[-\.]pre|-next|(a|b|c)[0-9]+|snapshot|master)" |
+    sed "s/^\s\+//" |
```

Comment:
@klane This was the fix on my mac. `sed` was only stripping the whitespace on the first line, leaving leading whitespace on the rest of the lines

```suggestion
    awk '{ print $1 }' |
```

Reply:
Thanks @homburg. That did the trick.

Reply:
Great 👍 I guess you have a brew installed `sed` if it worked differently on your machine.

What do you get for `which sed`?

Reply:
Yep I installed `sed` with Homebrew, which probably explains why the Linux tests passed.